source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
+}
+
+.figure__image {
+  @include block-spacing($end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
```

Comment:
eLife used `outline`, means there's no inline borders when full width.

Reply:
Ah, okay. Didn't realise `outline` was intentional (only usually use it for debugging), makes sense.